lib/util/AuthConfig.js

```diff
@@ -111,9 +120,64 @@ class AuthConfig extends Config {
             this.loginKey = undefined;
             this.refreshToken = undefined;
         }
+
+        var tempJSON = this._getAuthJSON();
+        
+
+        if(tempJSON == undefined) {
+            tempJSON = {};
+        }
+
+        if(Object.values(this._json)[0].isDefault) {
+            for (var account in tempJSON) {
+                if(tempJSON[account].isDefault){
+                    delete tempJSON[account].isDefault;
+                }
+            }
+        }
+        
+        const accountID = Object.keys(this._json)[0];
```

Comment:
Don't need this line?

Reply:
we don't? I needed it to get the accountID of the newly logged in account!